spce/ibi_gromacs-multi/settings.xml

```diff
@@ -0,0 +1,63 @@
+<cg>
+  <!-- example for a non-bonded interaction entry -->
+  <non-bonded>
+    <!-- name of the interaction -->
+    <name>CG-CG</name>
+    <!-- types involved in this interaction -->
+    <type1>CG</type1>
+    <type2>CG</type2>
+    <!-- dimension + grid spacing of tables for calculations -->
+    <min>0.0</min>
+    <max>0.9</max>
+    <step>0.01</step>
+    <inverse>
+      <!-- target distribution (rdf), just give gromas rdf.xvg -->
+      <target>CG-CG.dist.tgt</target>
+      <!-- update cycles -->
+      <do_potential>1</do_potential>
+      <!-- additional post processing of dU before added to potential -->
+      <post_update></post_update>
+      <!-- additional post processing of U after dU added to potential -->
+      <post_add></post_add>
+      <!-- name of the table for gromacs run -->
+      <gromacs>
+        <table>table_CG_CG.xvg</table>
+      </gromacs>
+    </inverse>
+  </non-bonded>
+  
+  <!-- general options for inverse script -->
+  <inverse>
+    <!-- 300*0.00831451 gromacs units -->
+    <kBT>2.49435</kBT>
+    <!-- use gromacs as simulation program -->
+    <program>gromacs</program>
+    <!-- gromacs specific options -->
+    <gromacs>
+      <!-- trash so many frames at the beginning -->
+      <equi_time>20</equi_time>
+      <!-- grid for table*.xvg !-->
+      <table_bins>0.002</table_bins>
+      <!-- cut the potential at this value (gromacs bug) -->
+      <pot_max>1000000</pot_max>
+      <!-- extend the tables to this value -->
+      <table_end>2.0</table_end>
+      <mdrun>
+        <command>mpirun -np 4 mdrun_mpi</command>
+        <opts>-multi 2</opts>
```

Comment:
Wouldn't it be good to let the user specify the name of the . mdp .top. files etc ? 
Because I think it's more helpful to learn how the .xml file changes in that context.

Reply:
In this example see use the same mdp, top etc for all simulations! I would only list them if they are different, like in the remd example.

Reply:
Got it.